md/release-notes.md

```diff
@@ -35,27 +53,50 @@ Extract informations from a project sources to generate an asciidoc document. Th
 
-#### UIPath connector: Cloud support
+#### UIPath connector - Cloud support
 The UIPath connector now supports the cloud solution.
 
 #### Collaboration - Select the branches to clone
-When cloning a project, gain time by selecting the branches you want to clone instead cloning all the branches.
+When cloning a project, gain time by selecting the branches you want to clone instead of cloning all the branches.
 
 #### Description field on widgets
 Add documentation to your pages by providing information on a widget (e.g. how to use it), thanks to the new Description field. Use this information to [generate documentation](release-notes.md#project-documentation).
+ 
+#### Bonita Platform Extensibility: Connectors and Actor filters archetypes
+You can now develop and test custom connectors and actor filters independently of the Bonita Studio.
+Bonita offers maven archetypes to help bootstrap these extension points
 
+More information:
+* [Bonita actor filter archetype](actor-filter-archetype.md)
+* [Bonita connector archetype](connector-archetype.md)
 
-### Runtime
+::: info
+This is not limited to 7.11 and can be used from version 7.7
+:::
 
-#### lib upgrade
-- spring, spring-boot
-- hibernate 4 to 5
-- ...
+### Runtime changes
 
+#### Monitoring
+##### Messages
+Two new message metrics are now accessible for monitoring purposes:
+* Potential: total number of potential couples, prior to duplicates removal. If this counter grows faster than the executed one, this indicates that we have lot of duplicates. See bonita.log for ids of duplicate couples.
+* ReTriggerMatchingTasks: number of times we detect that there are more potential matches to be processed.
+We retrieve the matches by batch of 100, a retrigger is performed if there are remaining potential matches. This counter increases when they are remaining messages to be processed. If this counter continues to increase, this means that we are continuously processing matches (even if no new waiting events and/or messages have been created), so events processing is delayed ie new events/messages are not processed on the fly as it should in a nominal situation.
+
+##### Connectors
+There is now a configurable time threshold on connectors. When connector execution takes longer than the threshold, a warning is logged. It allows identification of slow running services 
+The parameter is :
+```
+bonita.tenant.connector.warnWhenLongerThanMillis
+```
+
+#### Bonita Purge Tool
+A new open source project is available [here](https://github.com/bonitasoft/bonita-purge-tool)
+It cleans archived process instances older than a specific date. Allows to (potentially) reduce the size of a production database.
 
 ## Bundle changes
 
-Upgrade Tomcat from 8.5.47 to 8.5.53 (tomcat-dbcp from 9.0.16 to 9.0.31) **subject to change prior GA**
+Upgrade Tomcat from 8.5.47 to 8.5.53 (tomcat-dbcp from 9.0.16 to 9.0.31)
```

Comment:
For me, it should go to "Technical updates".

Reply:
It is Technical updates of bundle not Bonita, that is why is in this section. Added a new section Technical updates to Bundle section